dev/io.openliberty.microprofile.telemetry.1.0.internal.jaeger_fat/fat/src/io/openliberty/microprofile/telemetry/internal/utils/TestConstants.java

```diff
@@ -14,6 +14,11 @@
 
 public class TestConstants {
 
+    /**
+     * Environment variable to disable/enable the OTLP receiver
+     */
+    public static final String ENV_COLLECTOR_OTLP_ENABLED = "COLLECTOR_OTLP_ENABLED";
+
```

Comment:
I don't think this constant is needed (see my comment at the point where it's used) so it should be removed.

If it turns out that this constant is needed, then the copyright date on this file should be updated.